src/pages/ProfilesExplorerView/domain/variables/FiltersVariable/__tests__/filter-ops.spec.ts

```diff
@@ -0,0 +1,186 @@
+import { clearLabelValue, excludeLabelValue, includeLabelValue } from '../filters-ops';
+
+const FILTER_VEHICLE_IS_BIKE = Object.freeze({ key: 'vehicle', operator: '=', value: 'bike' });
+
+describe('includeLabelValue(filters, filterForInclude)', () => {
+  test.each([
+    [
+      'empty filters', // test name
+      [], // filters
+      FILTER_VEHICLE_IS_BIKE, // filterForInclude
+      [{ key: 'vehicle', operator: '=~', value: 'bike' }], // expected filters
+    ],
+    [
+      'filters with a different label',
+      [{ key: 'region', operator: '=~', value: 'eu-north' }],
+      FILTER_VEHICLE_IS_BIKE,
+      [
+        { key: 'region', operator: '=~', value: 'eu-north' },
+        { key: 'vehicle', operator: '=~', value: 'bike' },
+      ],
+    ],
+    [
+      'filters with the "in" operator and the same label but different value',
+      [{ key: 'vehicle', operator: '=~', value: 'car' }],
+      FILTER_VEHICLE_IS_BIKE,
+      [{ key: 'vehicle', operator: '=~', value: 'car|bike' }],
+    ],
+    [
+      'filters with the "in" operator and the same label+value #1',
+      [{ key: 'vehicle', operator: '=~', value: 'bike' }],
+      FILTER_VEHICLE_IS_BIKE,
+      [{ key: 'vehicle', operator: '=~', value: 'bike' }],
+    ],
+    [
+      'filters with the "in" operator and the same label+value #2',
+      [{ key: 'vehicle', operator: '=~', value: 'car|bike' }],
+      FILTER_VEHICLE_IS_BIKE,
+      [{ key: 'vehicle', operator: '=~', value: 'car|bike' }],
+    ],
+    [
+      'filters with the "not in" operator and the same label but different value',
```

Comment:
**note:** UI behaves exactly like the test -> nothing changes, but the description of the tests says the value should be different